apps/storefront/vite.config.ts

```diff
@@ -65,7 +66,11 @@ export default defineConfig(({ mode }) => {
         reporter: ['text', 'html', 'clover', 'json'],
       },
       deps: {
-        inline: ['react-intl'],
+        optimizer: {
+          web: {
+            include: ['react-intl'],
+          },
+        },
       },
       poolOptions: {
         threads: {
```

Comment:
Could you try to enable the parallel test too? It was disabled cause an error, but it might worth to check if the error is gone with this new update

Reply:
sure can do

Reply:
done, seems to work fine for now 🤷